Lib/tkinter/__init__.py

```diff
@@ -759,6 +759,7 @@ def after_idle(self, func, *args):
         Return an identifier to cancel the scheduling with
         after_cancel."""
         return self.after('idle', func, *args)
+
```

Comment:
It seems like this is an unrelated change.

Reply:
Adding the missing space between the two existing 'after' functions when adding a third, with the required space, seems fine with me.